Jenkinsfile

```diff
@@ -30,10 +29,7 @@ pipeline {
                     echo "Domain name is ${DOMAIN_NAME}"
 
                     echo '*#*#*#*#*#*#*#*#*#*#*#*#  Certificates Expiring Soon  *#*#*#*#*#*#*#*#*#*#*#*#*#*#*#'
-                    rc = sh(returnStatus: true, script: "keytool -list -v -keystore \'${pwd()}/nucleus/admin/template/src/main/resources/config/cacerts.jks\' -storepass \'changeit\' | grep -E \"Valid from:.*`date +'%b' --date='+1 month'`.*`date +'%Y'`|`date +'%b'`.*`date +'%Y'`\"")
-                    if(rc != 1) {
-                        currentBuild.result = 'UNSTABLE'
-                    }
+                    sh """keytool -list -v -keystore '${pwd()}/nucleus/admin/template/src/main/resources/config/cacerts.jks' -storepass 'changeit' | grep -E 'Valid from:.*`date +'%b' --date='+1 month'`.*`date +'%Y'`|`date +'%b'`.*`date +'%Y'`"""
```

Comment:
Can be removed also? I don't think anyone pays attention to this

Reply:
I even forgot about it

Reply:
Could but I'm just reverting the commit here, not doing any manual intervention

Reply:
Keep the scripts clean, might as well

Reply:
Looks like I'll have to, causes jenkins scripts to fail